.github/workflows/coccicheck.yaml

```diff
@@ -0,0 +1,12 @@
+name: BPF
+
+on:
+  - pull_request
+
+jobs:
+    coccicheck:
+        name: coccicheck
+        runs-on: ubuntu-18.04
```

Comment:
Nit: Looking at `go-mod.yaml`, it looks like we could also have `ubuntu-latest` instead of 18.04. Probably doesn't change much though.